### ISSUE TITLE
Orion Dev Epic: Email Notifications

### ISSUE THREAD

**Issue by dmtrjsg** [Joystream/orion#126]
@dmtrjsg commented on [Mon May 08 2023](https://github.com/Joystream/atlas/issues/4153)

**Comment by dmtrjsg** [comment]
1. Featuring video hero is a good idea "Your video is featured as the [category_title] category hero video." ✅ 
2. Same with Your NFT was featured on the marketplace "Your NFT [NFT_titile] is featured on the Marketplace." ✅ 
3. Yes, indeed, let's add Royalties "You received royalties for [NFT_title]." ✅ 
4. Channel followers, reactions to videos -> Probably an overkill you are right, great thinking, let's exclude.  ⛔ 

cc @ignazio-bovo @KubaMikolajczyk

**Comment by dmtrjsg** [comment]
@ignazio-bovo discussed again and decided to include channel followers and reactions to videos 😇 
cc @drillprop

**Comment by ignazio-bovo** [comment]
Just to point out that not all notifications listed have already handler/resolver implemented yet, is their implementation part of this scope yet? @dmtrjsg

**Comment by dmtrjsg** [comment]
> is their implementation part of this scope yet

Are you asking if the above is the full scope of events? - I'd say so.

@KubaMikolajczyk is including them to designs so my expectation is that Atlas team will handle them just as designed when backend is ready. cc @attemka

**Comment by ignazio-bovo** [comment]
what I mean is that for example if there's no `markChannelAsVerified` mutation in the first place there cannot be any notification deposit.
What I am asking is: in case of any of the above notifications miss handlers/resolvers needed to trigger them should I implement them in the scope of this sprint?

**Comment by dmtrjsg** [comment]
@ignazio-bovo Yes that makes sense @ignazio-bovo thank you for asking. The only one we are removing is the `password updated` notification 👍

**Comment by dmtrjsg** [comment]
MM

Agreed timeline:

User accounts need to be merged to Orion v3
- https://github.com/Joystream/orion/pull/122  to be merged as needed for user accs release.

V0 (dev) - support notification preferences for Atlas
Draft PR is opened and it contains this part > https://github.com/Joystream/orion/pull/156 
@ignazio-bovo to share the part covering the preferences with @thesan 

v1.  - circa 2 weeks from now
- All notifications where mappings are already created on Orion
- templates
* Manual testing on Atlas

@ignazio-bovo to share the list of notiffs

v1.1 - circa 2 more weeks
- Notifications where runtime event exists but there are no mappings on Orion
- email digest

V1.2  - timeline unclear as this one is tricky
- timed auction ended
- bid can be withdrawn


v1.3 
- Auto-tests

_________

**Comment by ignazio-bovo** [comment]
> * [ ]  Channel received transfer from WG
> * [ ]  New payout is claimable from Council Payout proposal

See https://github.com/Joystream/orion/issues/155 it contains a section with mappings that are already implemented, and the one that needs to be implemented (in v2)
It contains also a status on the template generation
